Libraries/Components/Touchable/TouchableHighlight.js

```diff
@@ -225,7 +275,7 @@ var TouchableHighlight = React.createClass({
 
   render: function() {
     return (
-      <View
+      <TVView
```

Comment:
I'd love to see an example of how you would like to modify the Touchable components for TV... I think the Platform.ios.js changes are good to have in any case, as they make it possible for developers to have common components with iOS and tvOS specific customizations or styles.